mlir/lib/Dialect/ArmSME/Transforms/OuterProductFusion.cpp

```diff
@@ -31,6 +31,55 @@ using namespace mlir;
 using namespace mlir::arm_sme;
 
 namespace {
+
+// Common match failure reasons.
+static constexpr StringLiteral
+    MATCH_FAILURE_NO_ACCUMULATOR("no accumulator operand");
```

Comment:
Aren't ALL_CAPS reserved for macros? Shouldn't these be camelCase? https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly

Reply:
Just an aside/observation: MLIR is not following the LLVM coding standards exactly, it's a mish-mash of Google style + LLVM style (but follows neither exactly), in both new and old code.

Reply:
E.g. MLIR is not following the LLVM rules for variable names (nor the Google one for that matter). But it does seem to follow Google constant naming rules.

Reply:
>Just an aside/observation: MLIR is not following the LLVM coding standards exactly, it's a mish-mash of Google style + LLVM style (but follows neither exactly), in both new and old code.

Yeah I noticed in that doc it says variables start with upper case but that's not true in MLIR. Anyhow, for this particular case, I can't see anywhere else that uses upper case for constexpr like this so I'll change to camel case given it's prevalent.